src/web/app/pages-instructor/instructor-session-result-page/instructor-session-no-response-panel.component.ts

```diff
@@ -34,6 +34,7 @@ export class InstructorSessionNoResponsePanelComponent implements OnInit, OnChan
   SortBy: typeof SortBy = SortBy;
   SortOrder: typeof SortOrder = SortOrder;
 
+  @Input() isNoStudentsLoaded: boolean = false;
```

Comment:
Not a good field/variable name. `NoStudentsLoaded` gives the impression of "no students are loaded", not "students with no responses are loaded".

Reply:
Sounds good, I changed it to follow the naming of the original `noResponseStudents`